lib/elixir/lib/uri.ex

```diff
@@ -375,15 +375,19 @@ defmodule URI do
   defdelegate to_string(uri), to: String.Chars.URI
 
   @doc ~S"""
-  Merges two URIs as per RFC 3986 §5.2
+  Merges two URIs.
+
+  This function merges two URIs as per [RFC
+  3986](https://www.ietf.org/rfc/rfc3986.txt) §5.2.
 
   ## Examples
 
-      iex> URI.merge(URI.parse("http://google.com"), "/query") |> to_string
+      iex> URI.merge(URI.parse("http://google.com"), "/query") |> to_string()
       "http://google.com/query"
 
-      iex> URI.merge("http://example.com", "http://google.com")|> to_string
+      iex> URI.merge("http://example.com", "http://google.com")|> to_string()
```

Comment:
Yes. Yesterday I spent 10 minutes trying to find a bug because a variable was accidentally being used as a function call, so those are the only harmful cases, the other cases aren't ambiguous. :)